Update licence keys section for pi 5 (#3560)
raspberrypi/documentation: documentation/asciidoc/computers/config_txt/codeclicence.adoc

Hardware decoding of additional codecs on the Raspberry Pi 3 and earlier models can be enabled by https://codecs.raspberrypi.com/license-keys/[purchasing a licence] that is locked to the CPU serial number of your Raspberry Pi.

> The Raspberry Pi 4 has permanently disabled hardware decoders for MPEG2 and VC1. These codecs cannot be enabled, so a hardware codec licence key is not needed. Software decoding of MPEG2 and VC1 files performs well enough for typical use cases.

Reviewer: I think it's better to not mention that these hardware blocks exist on the Pi 4 series; they also don't exist on Pi 5 series devices. Perhaps something like `Raspberry Pi 4 and 5 do not use hardware decode for MPEG2 and VC1; software decoding performs well enough for typical use cases.`

Author: This paragraph talks about the Pi 4, so I don't want to make things messy by talking about the 5 in the last sentence. IMO the cat's already out of the bag that these disabled decoders exist, since it's already public information in the docs. So we might as well acknowledge it so anybody who knows about those disabled decoders can find out that they're impossible to use, even with a licence.

@aallan does that seem reasonable to you? I can remove mention of the Pi 4 disabled decoders if you feel that it isn't useful. I just don't want to remove useful information that's already in the docs.

Reviewer 2: Keep it.

> The Raspberry Pi 5 has HEVC and H.265 hardware decoders. Both are enabled by default and do not require a licence key, so a hardware codec licence key is not needed.

Reviewer: HEVC **_is_** H.265.

Author: ```suggestion
The Raspberry Pi 5 has HEVC/H.265 hardware decoding. This decoding is enabled by default, so a hardware codec licence key is not needed.
```

The Raspberry Pi 4 has permanently disabled hardware decoders for MPEG2 and VC1. These codecs cannot be enabled, so a hardware codec licence key is not needed. Software decoding of MPEG2 and VC1 files performs well enough for typical use cases.

> The Raspberry Pi 5 has HEVC/H.265 hardware decoding. This decoding is enabled by default, so a hardware codec licence key is not needed.

Reviewer 2: I think that's generally written as HEVC (H.265). But YMMV.

Reviewer 2: I had no clue at all that H.264 was also known as AVC, so there's that! 🤣

Author: ```suggestion
The Raspberry Pi 5 has H.265 (HEVC) hardware decoding. This decoding is enabled by default, so a hardware codec licence key is not needed.
```
